src/test/java/com/kainos/ea/integration/JobRoleIntegrationTest.java

```diff
@@ -26,11 +37,50 @@ public class JobRoleIntegrationTest {
     );
 
     @Test
-    void getJobRoles_shouldReturnListOfJobRoles() {
-        List<JobRole> response = APP.client().target("http://localhost:8080/api/job-roles")
+    void getJobRoles_shouldReturn200_whenValidRoleIsPassedIn () {
+
+        String fakeSecret = System.getenv("JWT_SECRET");
+        String expectedUsername = "ExpectedUsername";
+
+        Key hmacKey = new SecretKeySpec(Base64.getDecoder().decode(fakeSecret),
+                SignatureAlgorithm.HS256.getJcaName());
+
+        String token = Jwts.builder()
+                .claim("username", expectedUsername)
+                .claim("role", "View")
+                .setExpiration(DateUtils.addHours(new Date(), 1))
+                .signWith(hmacKey)
+                .compact();
+
+        Response response = APP.client().target("http://localhost:8080/api/job-roles")
+                .request()
+                .header("Authorization", token)
+                .get();
+
+        Assertions.assertEquals(200, response.getStatus());
+    }
+
+    @Test
+    void getJobRoles_shouldReturn401_whenInvalidRoleIsPassedIn () {
+
+        String fakeSecret = System.getenv("JWT_SECRET");
```

Comment:
fakeSecret that isn't fake

Reply:
shshs - > They will never know. All fixed :)